packages/core/src/extensions/Blocks/nodes/HeadingBlock.ts

```diff
@@ -0,0 +1,65 @@
+import { Node, NodeViewRendererProps } from "@tiptap/core";
+import styles from "./Block.module.css";
+
+export type HeadingBlockAttributes = {
+  level: string;
+};
+
+export const HeadingBlock = Node.create({
+  name: "headingBlock",
+  content: "inline*",
+
+  addAttributes() {
+    return {
+      level: { default: "1" },
+    };
+  },
+
+  addNodeView() {
+    return (props: NodeViewRendererProps) => {
+      const element = document.createElement("div");
+      element.setAttribute("data-node-type", "block-content");
```

Comment:
Honestly I need to see if we even check for `block-content` anywhere. This and the `content-type` attributes probably need a bit of cleaning up, since it's likely we don't need both of them.

Reply:
So it seems like both attributes are only really uesful for styling, might be good to discuss later what the plan is for that.

Reply:
Another thing to add is that `block-content` isn't that useful even for styling since `.block > div:first-child` also works, though at the expense of readability.

Reply:
Styling uses classnames, I think `data-node-type` is only used for testing right? **if** that's the case I'd rather have "data-test" as prefix